airflow/providers/common/sql/operators/sql.py

```diff
@@ -46,18 +53,10 @@ def parse_boolean(val: str) -> str | bool:
     raise ValueError(f"{val!r} is not a boolean-like string value")
 
 
-def _get_failed_checks(checks, col=None):
-    if col:
-        return [
-            f"Column: {col}\nCheck: {check},\nCheck Values: {check_values}\n"
-            for check, check_values in checks.items()
-            if not check_values["success"]
-        ]
-    return [
-        f"\tCheck: {check},\n\tCheck Values: {check_values}\n"
-        for check, check_values in checks.items()
-        if not check_values["success"]
-    ]
+def _raise_exception(exception_string: str, retry_on_failure: bool) -> NoReturn:
+    if retry_on_failure:
+        raise AirflowException(exception_string)
+    raise AirflowFailException(exception_string)
```

Comment:
Since this function is always used with `self.retry_on_failure`, perhaps it should be a method instead and we can get rid of the second argument?